src/openrct2-ui/windows/Options.cpp

```diff
@@ -970,13 +970,15 @@ static void window_options_mousedown(rct_window *w, rct_widgetindex widgetIndex,
             config_save_default();
             gfx_invalidate_screen();
             context_trigger_resize();
+            context_update_cursor_scale();
```

Comment:
there is one more place where it needs to get called – in handler for `set window_scale`